### ISSUE TITLE
Can I use my Home Page as the Membership Options Page?

### ISSUE THREAD

**Issue by raamdev** [websharks/s2member-kb#159]
---
##### KB Article Creation Checklist

- [x] Write initial draft for this KB Article; label this issue `draft` and either `questions` or `tutorials`
- [x] Add [required YAML configuration](https://github.com/websharks/team/wiki/KB-Articles-::-Required-YAML-Configuration)
- [x] Add Tags for this KB Article to the YAML config (see [YAML Keys (Explained)](https://github.com/websharks/team/wiki/KB-Articles-::-Required-YAML-Configuration#yaml-keys-explained))
- [x] Edit and finalize draft for publishing (remove `draft` label, add `draft-finalized` label)
- [x] Assign Issue to yourself and [create Markdown file](https://github.com/websharks/team/wiki/KB-Articles-::-Creating-the-Markdown-File) (remove `draft-finalized` label, add `pending`)
- [x] Project Lead: Review and Publish KB Article (remove `pending` label, add `published` label)

##### Additional TODOs

_Additional TODO list items go here._

---

**Markdown File:** http://wsharks.com/18RjY88
**Published KB Article:** http://s2member.com/kb-article/can-i-use-my-home-page-as-the-membership-options-page/

---

**Comment by raamdev** [comment]
@jaswsinc Referencing related internal ticket: https://websharks.zendesk.com/agent/tickets/5536

I'm not seeing any elegant way of doing this. Even doing something like redirecting a MOP to the Home Page wouldn't work because that would make MOP vars not work (unless we somehow forward all query vars to the home page when redirecting...?).

Is there any way to make this work? What suggestions do you have?

**Comment by jaswsinc** [comment]
> I'm not seeing any elegant way of doing this.

Right. I have looked at this in the past, and while it _is_ a limitation so-to-speak, there's really very little reason to use your Home Page as the Membership Options Page anyway. And in fact, doing so causes hiccups for several conditional tags that s2Member analyzes which can lead to unanticipated problems, because when a "Post" (i.e., a Page) is reclassified as the Home Page in "the WordPress way", this changes some of the WordPress Conditional Tags when analyzing that content, which s2Member depends on in several places.

Long story short, we _could_ pull this off, but I don't see it as a priority issue. When site owners realize that a Membership Options Page is there mostly to serve as a redirect for unprivileged users, it becomes obvious that you really can put your Payment Buttons, Pricing Options, Pro-Forms, etc... anywhere you like. If you want them on the home page that's perfectly understandable and OK. However, you still need to configure a Membership Options Page for s2Member that is dedicated to that purpose.

**Comment by tracyselena** [comment]
I've read this post....my question is ... ok so I have a home page with mypaypal button on it b/c it's my sales page... I've also got a Member Options page.. can that be blank? As long as I indicate it's the members page in 2smember general settings? OR do my users have to use that page to buy their membership?

**Comment by jaswrks** [comment]
@tracyselena It can be blank, just needs to exist. You can put buttons/forms on any other page that you like, or just link to other pages from your Membership Options Page.
